ui/src/literature/containers/DetailPageContainer/DetailPageContainer.jsx

```diff
@@ -149,9 +149,9 @@ function DetailPage({
                         text="links"
                         trackerEventId="LiteratureFileLink"
                       />
-                    )}
-                    {displayClaimingButton()}
+                    )}  
                     <CiteModalActionContainer recordId={controlNumber} />
+                    {displayClaimingButton()}
```

Comment:
it doesn't require updating the snapshot?

Reply:
there is no test for this component 🙈

Reply:
in cypress I mean

Reply:
In cypress the snapshot is ok, but I added unit test :)

Reply:
The tests probably won't fail cause the difference is pretty small, but maybe it would be worth having an updated view for this component?

Reply:
You are right - snapshots updated 🫡